### ISSUE TITLE
Project Feedback!

### ISSUE THREAD

**Issue by codepathreview** [maxinenzegwu/Flixster#1]
It looks like your video walkthrough doesn't seem to show the details of the movie's popularity. Can you update this and resubmit?

**Comment by maxinenzegwu** [comment]
Hello!
I'm a little confused with this comment. I just rewatched my video
walkthrough and it shows the horizontal backdrop image in landscape mode
and the tall poster image in portrait mode. Is that not what you mean?
Thank you,
Maxine

**Comment by codepathreview** [comment]
Yes, that's correct that you have those in your video walkthrough. All we need to see is the details of the movie's popularity along with the ratings. This is one of the required user stories, you can refer to #4 in the Assignment tab. Let me know if you have further clarifications. 
/cc @codepathreview

**Comment by maxinenzegwu** [comment]
I believe this was also shown in the walkthrough I submitted. I thought you
said that I didn’t have the poster image appear in landscape mode? Can you
please confirm which of these is the problem because I believe my
walkthrough shows both of these.

**Comment by maxinenzegwu** [comment]
I see the issue now after speaking with a TA, I've just resubmitted

**Comment by maxinenzegwu** [comment]
Hello, I've resubmitted after help with a TA and believe the issue is
fixed. Can you confirm?

Thank you,
Maxine

**Comment by codepathreview** [comment]
Nice work, Maxine!

One of the most important parts of these projects is that you add additional features and tweak the UI / UX because that will provide many more learning opportunities. I encourage you to complete the projects each week with required stories early and then spend time adding your own UI elements and experimenting with stretch features that will improve the user experience.

We have provided a detailed Flixster Feedback Guide here: https://courses.codepath.org/snippets/android_university_fast_track/feedback_guides/project_2_feedback.md, which covers the most common points we see for this project. Read through the feedback guide point-by-point to determine other ways you could improve your submission. You should consider going back and implementing applicable feedback as well. Keep in mind that one of the most important parts of Android development is learning the correct patterns and conventions.

Hopefully this project has given you a better sense of working with RelativeLayout which is a very flexible layout system, probably one of the most powerful responsive-first layout systems available across web and mobile platforms, and RecyclerView, which is a superior stand-in for the ListView we used in the Todo project. This assignment also gave us our first introduction to networking, working with APIs and loading remote images. The next assignment will introduce new concept such as accepting user input and navigating between activities but will also reinforce important concepts such as networking, using APIs, and handling remote images.

If you have any particular questions about the assignment, feel free to reply here or or email us support@codepath.org. 
/cc @codepathreview
